src/test/ui/print_type_sizes/padding.stdout

```diff
@@ -0,0 +1,21 @@
+print-type-size type: `E1` overall bytes: 24 align: 8
+print-type-size    discrimimant bytes: 4
+print-type-size    variant A exact bytes: 16
+print-type-size        padding bytes: 4
+print-type-size        field .0 bytes: 8 align: 8
```

Comment:
AIUI rustc is going to be able to rearrange field order in the future to improve packing. In that case would the field numbers be listed out of order?